website/source/docs/platform/k8s/service-sync.html.md

```diff
@@ -53,7 +53,26 @@ syncCatalog:
 
 This will enable services to sync _in both directions_. You can also choose
 to only sync Kubernetes services to Consul or vice versa by disabling a direction.
-See the [Helm configuration](/docs/platform/k8s/helm.html#configuration-values-)
+
+To only enable syncing Consul services to Kubernetes use the config:
+
+```yaml
+syncCatalog:
+  enabled: true
+  toConsul: false
+  toK8S: true
+```
+
+To only enable syncing Kubernetes services to Consul use:
+
+```yaml
+syncCatalog:
+  enabled: true
+  toConsul: true
+  toK8S: false
+```
+
+See the [Helm configuration](/docs/platform/k8s/helm.html#v-synccatalog)
 for more information.
 
 -> **Before installing,** please read the introduction paragraphs for the
```

Comment:
This note feels a little bit out of place, and I'd love to find a way to put it above the installation instructions.